tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "forceConsistentCasingInFileNames": true,
     "types": ["node", "jest", "@edge-runtime/types"],
     "noImplicitAny": false,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "moduleResolution": "Node"
```

Comment:
Ensures compatibility between CommonJS and ESModules syntax.

Reply:
From what I'm reading , this now ensures that Typescript's module-resolution algorithm is `Node` (as opposed to `Classic`). 

`Node` vs `Classic` have different 'rule sets' Typescript assumes in order to provide correct `types` on imports at runtime (lifted from [here](https://www.typescriptlang.org/docs/handbook/modules/theory.html#esm-and-cjs-interoperability)).